### ISSUE TITLE
Classification results error

### ISSUE THREAD

**Issue by lixun95** [kratzert/finetune_alexnet_with_tensorflow#66]
Thanks for your perfect code!
when I finished my training process, I want to get the result(s) after input my test image(s) using a python test script, my dataset num_classes is 10, but the output results only have 3 classes. This means no matter what class of image I input, the model will classify them into the first 3 classes. For example, my dataset class_name is:
`class_name = ['BACT', 'BYST', 'CAOX', 'HYAL', 'MUCS', 'RBC', 'SPRM', 'SQEP', 'WBC', 'WBCC']`
but when I input 'BACT', 'BYST' and 'CAOX' (the first 3 classes), the model will output the correct results while I input other classes images just like 'MUCS', it will also output one of the first 3 classes results.
And this is my testing process screenshot:
![test2](https://user-images.githubusercontent.com/32358982/46869311-e986b500-ce5d-11e8-9165-9db7326f881f.png)
I am sure that my training num_classes and my testing num_classes are the same 10, and my output is softmax connected to fc8. Can you tell me what's wrong with my codes?

This is my test script:
```
import tensorflow as tf
from alexnet import AlexNet  
import time

class_name = ['BACT', 'BYST', 'CAOX', 'HYAL', 'MUCS', 'RBC', 'SPRM', 'SQEP', 'WBC', 'WBCC']  
path_image = "/home/lixun/Documents/Urine/finetune_alexnet_with_tensorflow-master/test_image/MUCS-1201.jpeg"
num_class = 10
model_path = "/home/lixun/Documents/Urine/finetune_alexnet_with_tensorflow-master/checkpoints/model_epoch10.ckpt"

def test_image(path_image, num_class, weights_path='Default'):
    img_string = tf.read_file(path_image)
    img_decoded = tf.image.decode_jpeg(img_string, channels=3)
    img_resized = tf.image.resize_images(img_decoded, [227, 227])
    img_resized = tf.reshape(img_resized, shape=[1, 227, 227, 3])

    model = AlexNet(img_resized, 0.5, num_class, skip_layer='', weights_path=weights_path)
    score = tf.nn.softmax(model.fc8)
    max = tf.arg_max(score, 1)
    saver = tf.train.Saver()

    with tf.Session() as sess:
        since = time.time()
        sess.run(tf.global_variables_initializer())
        saver.restore(sess, model_path)  
        # score = model.fc8
        print(sess.run(model.fc8))
        prob = sess.run(max)[0]
        elapsed = time.time() - since
        print("Finished in {} minutes and {} seconds".format(elapsed // 60, elapsed % 60))
        print("Class:" + class_name[prob])

test_image(path_image, num_class)  
```
